.github/pull_request_template.md

```diff
@@ -1,10 +1,11 @@
+<!-- 
 Hi! Thanks for contributing to JupyterLab.
 Please look through the checklist below before submitting your pull request.
+-->
 
 - [ ] Look through open pull requests to see if someone has already started working on this.
-- [ ] Note what issue(s) this pull request addresses. (We strongly encourage the community to open an issue describing proposed work before opening a pull request.)
-- [ ] Provide a detailed description of the pull request. (Why are these changes necessary? How have they been implemented?)
-- [ ] For UI/UX changes, include before/after screenshots
-- [ ] Include a narrative description of visual or user interaction changes. How does your design effectively address the problem?
-- [ ] Provide a list of Jupyterlab packages the pull request modifies.
-- [ ] Describe any backwards incompatible changes to Jupyterlab’s public APIs.
+- [ ] Note what issue(s) this pull request addresses.
+- [ ] Describe the changes and reasons for them in this pull request.
+- [ ] Describe visual or user interaction changes in sentences. How does your design effectively address the problem?
```

Comment:
How does "the design"?  This seems to be accusatory.

Reply:
better?